frontend/cypress/integration/utils/psychologist.js

```diff
@@ -17,7 +17,11 @@ const suspend = () => {
   });
 };
 
-const signConvention = (name, isSigned) => cy.request('GET', 'http://localhost:8080/api/university')
+const signConvention = (name, isSigned) => cy.request({
+  method: 'GET',
+  url: 'http://localhost:8080/api/universities',
+  headers: { 'xsrf-token': 'randomXSRFToken' },
```

Comment:
this is working ?

Reply:
I don't know what you mean exactly, but tests are passing yes.